src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/util/ProcessUtil.java

```diff
@@ -297,20 +297,32 @@ public static boolean isWindowsProcessAlive(InstanceConfiguration config, String
     {
         Map<String, String> result = null;
         
+        try
+        {
+            result = EnvironmentUtils.getProcEnvironment();
+        }
+        catch (IOException ex)
+        {
+            throw new ElasticsearchSetupException(
+                    "Cannot get the current process environment", ex);
+        }
+
         if (environment != null)
         {
-            try
-            {
-                result = EnvironmentUtils.getProcEnvironment();
-            }
-            catch (IOException ex)
-            {
-                throw new ElasticsearchSetupException(
-                        "Cannot get the current process environment", ex);
-            }
             result.putAll(environment);
         }
-        
+
+        // the elasticsearch start/plugin scripts print warnings if these environment variables are passed
+        // and unsets these. And because the scripts would print a warning, we can't rely on the output :(
+        if(result.containsKey("JAVA_TOOL_OPTIONS"))
```

Comment:
Since `remove` does nothing if there's no value for the given key, the `if` is not required; same comment on line 321. The code will be cleaner without the if's.

Reply:
👍 done...